app.py

```diff
@@ -40,7 +40,8 @@
     'components.handlers.students_affected_by_module.StudentsAffectedByModule',
     '/addModule', 'components.handlers.add_module_handler.AddModule',
     '/moduleTakenPriorToOthers', 'components.handlers.module_taken_prior_to_others.TakePriorTo',
-    '/nonOverlappingModules', 'components.handlers.non_overlapping_modules.NonOverlappingModules'
+    '/nonOverlappingModules', 'components.handlers.non_overlapping_modules.NonOverlappingModules',
+    '/moduleTakenPriorToInternship', 'components.handlers.modules_taken_prior_to_internship.TakePriorInternship'
```

Comment:
I think this pylint issue should be ignored to make reading and modifying of the routes more easily.